### ISSUE TITLE
[auto-generated] warning: TeamCityExUnitFormatting does not know how to process event 

### ISSUE THREAD

**Issue by bozydar** [KronicDeth/intellij-elixir#1003]
```
warning: TeamCityExUnitFormatting does not know how to process event ({:module_started, %ExUnit.TestModule{name: CredoCheckCase, state: nil, tests: []}}).  
  /tmp/intellij_elixir/exunit/team_city_ex_unit_formatting.ex:189: TeamCityExUnitFormatting.put_event/2
  /tmp/intellij_elixir/exunit/1.4.0/team_city_ex_unit_formatter.ex:11: TeamCityExUnitFormatter.handle_cast/2
  (stdlib) gen_server.erl:616: :gen_server.try_dispatch/4
  (stdlib) gen_server.erl:686: :gen_server.handle_msg/6
  (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
```

**Comment by AndroidOatmeal** [comment]
I was having this message as well after upgrading to elixir 1.6

**Comment by KronicDeth** [comment]
Fixed by #1021.  Upgrade to 7.3.0.
